### ISSUE TITLE
uiquery will return nil and cause NoMethodError when invoking touch

### ISSUE THREAD

**Issue by mcspring** [calabash/calabash-ios#801]
### Project settings
<img width="688" alt="screen shot 2015-07-11 at 2 26 00 pm" src="https://cloud.githubusercontent.com/assets/7650/8632696/228158d0-27d9-11e5-9866-c8023a2ce440.png">

### calabash-ios console
```bash
touch("view marked:'News'")
NoMethodError: undefined method `to_sym' for nil:NilClass
	from /Users/MC/.rvm/gems/ruby-2.2.0@testing/gems/calabash-cucumber-0.14.3/lib/calabash-cucumber/actions/instruments_actions.rb:122:in `normalize_rect_for_orientation!'
	from /Users/MC/.rvm/gems/ruby-2.2.0@testing/gems/calabash-cucumber-0.14.3/lib/calabash-cucumber/actions/instruments_actions.rb:115:in `find_and_normalize'
	from /Users/MC/.rvm/gems/ruby-2.2.0@testing/gems/calabash-cucumber-0.14.3/lib/calabash-cucumber/actions/instruments_actions.rb:86:in `query_action'
	from /Users/MC/.rvm/gems/ruby-2.2.0@testing/gems/calabash-cucumber-0.14.3/lib/calabash-cucumber/actions/instruments_actions.rb:15:in `touch'
	from /Users/MC/.rvm/gems/ruby-2.2.0@testing/gems/calabash-cucumber-0.14.3/lib/calabash-cucumber/core.rb:1166:in `query_action_with_options'
	from /Users/MC/.rvm/gems/ruby-2.2.0@testing/gems/calabash-cucumber-0.14.3/lib/calabash-cucumber/core.rb:177:in `touch'
	from (irb):26
	from /Users/MC/.rvm/rubies/ruby-2.2.0/bin/irb:11:in `<main>'
```

### calabash-ios console with debug
```bash
touch("view marked:'News'")

[105, 114] in /Users/MC/.rvm/gems/ruby-2.2.0@testing/gems/calabash-cucumber-0.14.3/lib/calabash-cucumber/actions/instruments_actions.rb
   105:   end
   106:
   107:   # @!visibility private
   108:   def find_and_normalize(ui_query)
   109:     byebug
=> 110:     raw_result = raw_map(ui_query, :query)
   111:     orientation = raw_result['status_bar_orientation']
   112:     res = raw_result['results']
   113:
   114:     return res if res.empty?
(byebug) n

[106, 115] in /Users/MC/.rvm/gems/ruby-2.2.0@testing/gems/calabash-cucumber-0.14.3/lib/calabash-cucumber/actions/instruments_actions.rb
   106:
   107:   # @!visibility private
   108:   def find_and_normalize(ui_query)
   109:     byebug
   110:     raw_result = raw_map(ui_query, :query)
=> 111:     orientation = raw_result['status_bar_orientation']
   112:     res = raw_result['results']
   113:
   114:     return res if res.empty?
   115:
(byebug) raw_result
{"results"=>[{"class"=>"UILabel", "rect"=>{"center_x"=>288, "y"=>553, "width"=>64, "x"=>256, "center_y"=>558.5, "height"=>11}, "frame"=>{"y"=>36, "width"=>64, "x"=>0, "height"=>11}, "UIType"=>"UIView", "description"=>"<UILabel: 0x78e4e010; frame = (0 36; 64 11); text = '我的'; userInteractionEnabled = NO; layer = <_UILabelLayer: 0x78e4dfc0>>"}], "outcome"=>"SUCCESS"}
```

**Comment by jmoody** [comment]
We need more information.  Please see: [Reporting Problems](https://github.com/calabash/calabash-ios/wiki#reporting-problems).

I am pretty sure that your Calabash Server is out of date.

https://github.com/calabash/calabash-ios/wiki/Updating-your-Calabash-iOS-version

**Comment by jmoody** [comment]
Closing because of lack of response.  @mcspring I think you need to update your Calabash Server version.  Reopen this case if that does not solve your problem.

**Comment by mcspring** [comment]
Thanks! I'll try it again.

On Thu, Jul 23, 2015 at 2:35 PM, Joshua Moody <notifications@github.com>
wrote:

> Closed #801 <https://github.com/calabash/calabash-ios/issues/801>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/calabash/calabash-ios/issues/801#event-363086286>.
>



-- 
*Best Regards!*

*- Spring*
